source/Firmata/UwpFirmata.cpp

```diff
@@ -45,11 +44,12 @@ namespace {
 
 UwpFirmata::UwpFirmata(
     void
-):
+) :
     _data_buffer(new uint8_t[31]),
```

Comment:
now using a const, but the type of _data_buffer is std::unique_ptr which requires a pointer to be initialized